packages/next/server/dev/next-dev-server.ts

```diff
@@ -647,7 +657,9 @@ export default class DevServer extends Server {
     }
 
     if (!usedOriginalStack) {
-      if (type) {
+      if (type === 'warning') {
+        Log.warn(err + '')
```

Comment:
Why no "warning" prefix?

Reply:
@styfle The prefix is added by `Log`. The one that is interpolated in the line below is for `unhandledRejection` and `uncaughtException`. The message will show up prefixed.